apps/src/code-studio/pd/foorm/FoormLibraryEditor.jsx

```diff
@@ -39,27 +39,23 @@ const styles = {
 const PREVIEW_ON = 'preview-on';
 const PREVIEW_OFF = 'preview-off';
 
-class FoormEditor extends React.Component {
+class FoormLibraryEditor extends React.Component {
```

Comment:
for this component, I would combine these into one Editor component by making things a little more generic. The  `formCategories` and `libraryCategories` props could just be `categories`, and you could either get an object from redux that has the fields you need for each situation, or maybe accept all the props since none are required, with comments about when each will be there? I'm not sure what the ideal way to handle the props is.